### DISCUSSION TITLE
[#995486] Cursor makes glass render invisible

### DISCUSSION THREAD

**Discussion by xKingDark** [Mojang/minecraft-editor#69] [Bugs]
**How to Reproduce**:
  **1.** Place a glass block
  **2.** Hover over the glass with your cursor
  
**Observed Result**:
https://user-images.githubusercontent.com/47607185/224458473-4b86e229-c344-4f75-a857-c133c32ad045.MOV
https://user-images.githubusercontent.com/47607185/224484354-1c0893bc-0fab-4e51-adea-ef625793c4f5.mp4

**Expected Result**:
Glass shouldn’t become invisible

**Helpful Info**:
- Preview build version: `1.19.80.20`

**Comment by minecraftchloe** [comment]
Updated with internal tracking ID! Thank you!

**Comment by minecraftchloe** [comment]
Should be fixed! Let us know if this is an issue again!
